css/includes/components/_documentation.scss

```diff
@@ -154,14 +154,14 @@
     }
 }
 
-@include media-breakpoint-down(lg) {
```

Comment:
Unless I missed something this breakpoint is always superset by the `xxl` (previously `xl`) breakpoint.